web/src/enterprise/campaigns/detail/CampaignUpdateDiff.tsx

```diff
@@ -148,6 +149,9 @@ export const CampaignUpdateDiff: React.FunctionComponent<Props> = ({
             [_queryChangesets, campaign.id, _queryPatchesFromPatchSet, _queryPatchesFromCampaign, patchSet.id]
         )
     )
+    if (!campaign.viewerCanAdminister) {
+        return <HeroPage body="Cannot update campaign you don't have write-access to" />
```

Comment:
I suggest using the "permission levels" wording (from here https://github.com/sourcegraph/sourcegraph/pull/10921/files)

```suggestion
        return <HeroPage body="Updating a campaign is not permitted without campaign admin permissions" />
```